pkg/distributors/contracts/interfaces/IMultiDistributor.sol

```diff
@@ -33,48 +33,51 @@ interface IMultiDistributor {
         uint256 globalTokensPerStake;
     }
 
-    struct UserDistribution {
+    struct UserDistributionInfo {
         uint256 unclaimedTokens;
         uint256 userTokensPerStake;
     }
 
     struct UserStaking {
         uint256 balance;
         EnumerableSet.Bytes32Set subscribedDistributions;
-        mapping(bytes32 => UserDistribution) distributions;
+        mapping(bytes32 => UserDistributionInfo) distributionInfo;
     }
 
-    event Staked(bytes32 indexed distribution, address indexed user, uint256 amount);
-    event Unstaked(bytes32 indexed distribution, address indexed user, uint256 amount);
-    event DistributionCreated(
-        bytes32 indexed distribution,
+    event Staked(bytes32 indexed distributionChannel, address indexed user, uint256 amount);
+    event Unstaked(bytes32 indexed distributionChannel, address indexed user, uint256 amount);
+    event DistributionChannelCreated(
+        bytes32 indexed distributionChannel,
         IERC20 stakingToken,
         IERC20 distributionToken,
         address owner
     );
-    event DistributionDurationSet(bytes32 indexed distribution, uint256 duration);
-    event DistributionFunded(bytes32 indexed distribution, uint256 amount);
+    event DistributionDurationSet(bytes32 indexed distributionChannel, uint256 duration);
+    event DistributionFunded(bytes32 indexed distributionChannel, uint256 amount);
```

Comment:
I've left these as just `Distribution` as they are setting values for the next distribution although I'm a little torn on it. Thoughts?